polaris-service/build.gradle

```diff
@@ -1,218 +0,0 @@
-/*
- * Copyright (c) 2024 Snowflake Computing Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
-
-plugins {
-    alias(libs.plugins.shadow)
-    alias(libs.plugins.openapi.generator)
-    id("application")
-}
-
-dependencies {
-    implementation(project(":polaris-core"))
-    implementation(project(":polaris-eclipselink"))
-    
```

Comment:
@snazy , was it intentional that the `polaris-eclipselink` dependency wasn't included in the kotlin build? Or was that an oversight? I didn't see a mention of this in the PR description.

For changes to the build, I think a good practice would be to compare the contents of the bundled jars before and after to understand if there's a change in the build output

Reply:
Yea, my bad. That wasn't intentional.